modal.go

```diff
@@ -95,6 +95,17 @@ func (m *Modal) SetText(text string) *Modal {
 	return m
 }
 
+/// AddInputText adds buttons to the window. There must be at least one button and
```

Comment:
Following docklet might be better suited:

```suggestion
// AddInputText adds Input Fields to the window, which are later on identified by their labels.
```

Reply:
Looks good, thanks. Updating it.